package.json

```diff
@@ -1,37 +1,38 @@
 {
-	"name": "mirror-java",
-	"version": "1.0.0",
-	"description": "Mirror `Discord` Bot TypeScript edition",
-	"main": "built/index.js",
-	"scripts": {
-		"test": "echo \"Error: no test specified\" && exit 1",
-		"start": "node ./built/src/index.js"
-	},
-	"author": "Ford Friedel",
-	"license": "ISC",
-	"dependencies": {
-		"@discordjs/opus": "^0.8.0",
-		"@discordjs/voice": "latest",
-		"chrono-node": "latest",
-		"cron": "latest",
-		"discord-player": "latest",
-		"discord.js": "latest",
-		"enmap": "latest",
-		"ffmpeg": "latest",
-		"ffmpeg-static": "^4.4.1",
-		"geo-tz": "latest",
-		"mkdirp": "latest",
-		"node-cron": "latest",
-		"node-fetch": "^2.6.1",
-		"pm2": "^5.2.0",
-		"tslog": "^3.3.2",
-		"tweetnacl": "latest",
-		"typescript": "^4.5.5",
-		"ytdl": "latest",
-		"ytdl-core": "^4.10.0"
-	},
-	"devDependencies": {
-		"@types/mkdirp": "^1.0.2",
-		"@types/node-cron": "^3.0.1"
-	}
+    "name": "mirror-java",
```

Comment:
Please rebase your branch from main before merging, we should not have 8 files with changes that are not directly addressed in your PR.  Dont want to break production.